dev/source/docs/ros-distance-sensors.rst

```diff
@@ -0,0 +1,72 @@
+.. _ros-distance-sensors:
+
+==========================
+ROS distance sensors usage
+==========================
+
+.. warning::
+
+    Distance sensors message are currently support on :
+
+    - Ardupilot Master
+    - Copter >= 3.5.1
+    - Rover >= 3.2.0
+    - Plane >= 3.8.1
+
+
+Distance sensors message
+========================
+
+Ardupilot support both distance sensors message (`MAVLINK definition <http://mavlink.org/messages/common#DISTANCE_SENSOR>`__) both as input and output.
```

Comment:
Ardupilot support both distance sensors message -> ArduPilot support distance sensor message

I think that putting the link into *distance sensors message* would look better than having a parenthesis just saying *MAVLink definition*.